web-ui/src/main/resources/catalog/locales/en-core.json

```diff
@@ -563,5 +563,14 @@
     "metadataInfoReport": "Metadata information",
     "metadataErrorReport": "Metadata errors",
     "copyToClipboard": "Copy the text to the clipboard",
-    "textCopied": "Text has been copied to the clipboard."
+    "textCopied": "Text has been copied to the clipboard.",
+    "statusWorkflow-0": "Unknown",
```

Comment:
Status translation are coming from DB  https://github.com/geonetwork/core-geonetwork/blob/main/services/src/main/resources/config-spring-geonetwork.xml#L187 but not sure for id 21 and 24 ?

Reply:
That are special values handling the metadata that is approved and also has a working copy in draft or submitted. I can update to keep in this file only these values, sounds fine?

Reply:
21 and 24 are not in DB like https://github.com/geonetwork/core-geonetwork/blob/main/web/src/main/webapp/WEB-INF/classes/setup/sql/data/data-db-default.sql#L527-L552 right ? and only managed by Java when indexing ?

Reply:
Right, with the default statuses in the database, these cases that are relevant for the users to filter the metadata with a working copy:

- Approved with draft working copy
- Approved with submitted working copy

Otherwise, once the metadata is approved is not possible find if the metadata has the working copy submitted for review.